packages/aws-cdk/bin/cdk.ts

```diff
@@ -79,6 +79,7 @@ async function parseCommandLineArguments() {
       .option('bootstrap-customer-key', { type: 'boolean', desc: 'Create a Customer Master Key (CMK) for the bootstrap bucket (you will be charged but can customize permissions, modern bootstrapping only)', default: undefined, conflicts: 'bootstrap-kms-key-id' })
       .option('qualifier', { type: 'string', desc: 'Unique string to distinguish multiple bootstrap stacks', default: undefined })
       .option('public-access-block-configuration', { type: 'boolean', desc: 'Block public access configuration on CDK toolkit bucket (enabled by default) ', default: undefined })
+      .option('container-asset-scan-on-push', { type: 'boolean', desc: 'Scan container asset images on push on CDK toolkit ECR (disabled by default)', default: false })
```

Comment:
Why is this disabled by default?

Reply:
To match current behavior.

Reply:
opened a PR to enable by default:
https://github.com/aws/aws-cdk/pull/17994